tasks/components/layout/OrganizationDisplay.tsx

```diff
@@ -49,7 +49,7 @@ export const OrganizationDisplay = ({
   const { data } = useOrganizationsByUserQuery()
   const usedOrganizations: OrganizationDTO[] = organizations ?? data ?? []
 
-  const columns = width === undefined ? 2 : Math.max(Math.floor(width / 250), 1)
+  const columns = width === undefined ? 3 : Math.min(Math.floor(width / 250), 3)
```

Comment:
the thing is we dont support a width that is smaller than 250

Reply:
Yet the component should still show its content, although there might be display issues. For example on a screen that is barely below 250px, we should still try to display one column instead of nothing